tools/STOC/STOCeps2en.xml

```diff
@@ -0,0 +1,135 @@
+<tool id="stoc2" name="STOCeps2" version="1.0">
+    
+    <description>Annual variation and global trend by specialist groups from STOCeps (simple spot sampling).</description>
+    
+    <requirements>
+        <requirement type="package">r</requirement>
+        <requirement type="package">r-r.methodss3 </requirement>
+        <requirement type="package">r-arm</requirement>
+    </requirements>
+
+    <stdio>
```

Comment:
you can remove stdio and replace it with https://docs.galaxyproject.org/en/latest/dev/schema.html#detect-errors